images/prow-tests/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN gcloud components install docker-credential-gcr
 RUN docker-credential-gcr configure-docker
 
 # Replace kubectl with a working version
-# TODO(chizhg): remove once https://github.com/knative/test-infra/issues/1858 is fixed in releases <= 0.15 or those releases are no longer supported
+# TODO(chizhg): remove the pin after Knative release-0.19 is cut.
```

Comment:
you should keep the reasoning, but can add when it's expected to happen

Otherwise, anyone can look and this and say "why release-0.19" and worry whether it's really safe to remove

Reply:
Done.